modules/backend/lang/en/lang.php

```diff
@@ -9,6 +9,7 @@
         'invalid_type' => 'Invalid field type used :type.',
         'options_method_invalid_model' => "The attribute ':field' does not resolve to a valid model. Try specifying the options method for model class :model explicitly.",
         'options_method_not_exists' => "The model class :model must define a method :method() returning options for the ':field' form field."
+        'colors_method_not_exists' => "The model class :model must define a method :method() returning html color HEX codes for the ':field' form field."
```

Comment:
Please test run all code branches & relevant locales first: https://travis-ci.org/octobercms/october/jobs/417747998, you're missing a comma at the end of both of these lines